apps/arena/lib/arena/serialization/messages.pb.ex

```diff
@@ -483,7 +484,7 @@ end
 defmodule Arena.Serialization.KillEntry do
   @moduledoc false
 
-  use Protobuf, protoc_gen_elixir_version: "0.12.0", syntax: :proto3
+  use Protobuf, syntax: :proto3, protoc_gen_elixir_version: "0.12.0"
```

Comment:
I think you're using the wrong version of elixir, `syntax: :proto3` should be at the end when formatting it.